### DISCUSSION TITLE
Which version of MathNet supports the Universal Windows Platform?

### DISCUSSION THREAD

**Discussion by MOstanin** [mathnet/mathnet-numerics#820] [Q&A] [answered]

**Comment by cdrnet** [comment]
To my understanding UWP supports .Net Standard 2.0 now, so it should support any version >= v4.0 that way. Afaik older versions would still be supported but through the older PCL builds.

**Comment by songshizhao** [comment]
as i tried, new verson 4.9 is not supported in winstore by far (not supported API used),maybe 3.X is ok.

**Comment by cdrnet** [comment, marked as answer]
With the upcoming v5, the main library no longer includes any p/invoke code and should be much more compatible.
